x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -76,24 +76,26 @@ const AuthenticationTableComponent = pure<AuthenticationTableProps>(
     updateLimitPagination,
     type,
   }) => (
-    <LoadMoreTable
-      columns={getAuthenticationColumns()}
-      loadingTitle={i18n.AUTHENTICATIONS}
-      loading={loading}
-      pageOfItems={data}
-      loadMore={() => loadMore(nextCursor)}
-      limit={limit}
-      hasNextPage={hasNextPage}
-      itemsPerRow={rowItems}
-      updateLimitPagination={newLimit =>
-        updateLimitPagination({ limit: newLimit, hostsType: type })
-      }
-      title={
-        <h3>
-          {i18n.AUTHENTICATIONS} <EuiBadge color="hollow">{totalCount}</EuiBadge>
-        </h3>
-      }
-    />
+    <EuiPanel>
```

Comment:
Can we add the EuiPanel in the load more table so we do not have to change it everywhere and like that it will always be there

Reply:
thats a good idea, ill make the change